Build/loc/TranslationsImportExport.yml

```diff
@@ -18,45 +21,74 @@ schedules:
     - main
   always: true
 
-pool:
-  name: 'AzurePipelines-EO'
-  demands:
-  - ImageOverride -equals AzurePipelinesWindows2022compliant
-
-steps:
-- task: NodeTool@0
-  inputs:
-    versionSpec: '18.x'
-  displayName: 'Install Node.js'
-
-- task: CmdLine@2
-  inputs:
-    script: 'cd Extension && yarn install'
-
-- task: CmdLine@2
-  inputs:
-    script: 'cd ./Extension && yarn run translations-export && cd ..'
-
-- task: OneLocBuild@2
-  env:
-    SYSTEM_ACCESSTOKEN: $(System.AccessToken)
-  inputs:
-    locProj: 'Build/loc/LocProject.json'
-    outDir: '$(Build.ArtifactStagingDirectory)'
-    isCreatePrSelected: false
-    prSourceBranchPrefix: 'locfiles'
-    packageSourceAuth: 'patAuth'
-    patVariable: '$(OneLocBuildPat)'
-    LclSource: lclFilesfromPackage
-    LclPackageId: 'LCL-JUNO-PROD-VCPP'
-    lsBuildXLocPackageVersion: '7.0.30510'
-
-- task: CmdLine@2
-  inputs:
-    script: 'cd Extension && node ./translations_auto_pr.js microsoft vscode-cpptools csigs $(csigsPat) csigs csigs@users.noreply.github.com "$(Build.ArtifactStagingDirectory)/loc" vscode-extensions-localization-export/vscode-extensions && cd ..'
-
-- task: PublishBuildArtifacts@1
-  inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-    ArtifactName: 'drop'
-    publishLocation: 'Container'
+variables: 
+  TeamName: cpptools
+  Codeql.Language: javascript
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: AzurePipelinesWindows2022compliantGPT
+      os: windows
+    sdl:
+      sourceAnalysisPool: 
+        name: AzurePipelines-EO
+        image: AzurePipelinesWindows2022compliantGPT
+        os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
```

Comment:
You can delete the custom build tag. This was for tracking pipelines converted from classic using the tool, but it ended up getting copied everywhere. :)